# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/webcodecs/video-decoder.crossOriginIsolated.https.any.js.ini
# Commit: 68fae9bd7d92
# Full Hash: 68fae9bd7d921366d02d255a1fe6369848b6562d
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-29 15:31:45
# Regressor Bug: 1831451
# File Overlap Count: 1
# Description:
#   Bug 1831451 - Implement VideoDecoder::Configure r=padenot
#   
#   This patch implements configure method [1] for WebCodecs' VideoDecoder
#   interface. The configure job is via DecoderAgent's Configure().
#   
# ==============================================================================

diff -r 0fca9e6c2ef6 -r 68fae9bd7d92 testing/web-platform/meta/webcodecs/video-decoder.crossOriginIsolated.https.any.js.ini
--- a/testing/web-platform/meta/webcodecs/video-decoder.crossOriginIsolated.https.any.js.ini	Thu Jun 29 04:15:02 2023 +0000
+++ b/testing/web-platform/meta/webcodecs/video-decoder.crossOriginIsolated.https.any.js.ini	Thu Jun 29 04:15:02 2023 +0000
@@ -1,16 +1,10 @@
 [video-decoder.crossOriginIsolated.https.any.html]
-  [Test isConfigSupported() and configure() using a SharedArrayBuffer]
-    expected: FAIL
-
-  [Test isConfigSupported() and configure() using a Uint8Array(SharedArrayBuffer)]
-    expected: FAIL
+  prefs: [dom.media.webcodecs.enabled:true]
+  expected:
+    if (os == "android") and fission: [OK, TIMEOUT]
 
 
 [video-decoder.crossOriginIsolated.https.any.worker.html]
+  prefs: [dom.media.webcodecs.enabled:true]
   expected:
     if (os == "android") and debug and not swgl: [OK, TIMEOUT]
-  [Test isConfigSupported() and configure() using a SharedArrayBuffer]
-    expected: FAIL
-
-  [Test isConfigSupported() and configure() using a Uint8Array(SharedArrayBuffer)]
-    expected: FAIL